crates/kitsune_p2p/proxy/src/inner_listen.rs

```diff
@@ -71,14 +71,20 @@ pub async fn spawn_kitsune_proxy_listener(
         // Set up a timer to refresh our proxy contract at keepalive interval
         let i_s_c = i_s.clone();
         tokio::task::spawn(async move {
-            tokio::time::delay_for(std::time::Duration::from_millis(PROXY_KEEPALIVE_MS)).await;
-
-            if i_s_c.req_proxy(proxy_url.clone()).await.is_err() {
-                // either we failed because the actor is already shutdown
-                // or the remote end rejected us.
-                // if it's the latter - shut down our ghost actor : )
-                let _ = i_s_c.ghost_actor_shutdown().await;
+            loop {
+                tokio::time::delay_for(std::time::Duration::from_millis(PROXY_KEEPALIVE_MS)).await;
```

Comment:
let's bump this up to 30 seconds - we may also want a backlog item to delete the proxying record if we get an error trying to hook up a channel.

Reply:
https://www51.v1host.com/Holo/story.mvc/Summary?oidToken=Story:17568